src/include/elist.h

```diff
@@ -81,16 +96,21 @@ class elist {
 private:
   item _head;
   size_t item_offset;
+  ceph::atomic<size_t> items = {0};
```

Comment:
why `atomic`? using atomic this way enforces the sequential consistency between cores and could cause unnecessary pipeline stall.

Reply:
It seems make no sense to use the atomic, the user is responsible to make sure the list is not modified in parallel. I will change it back.